packages/runtime/src/widgets/Form/Dropdown.tsx

```diff
@@ -228,6 +228,7 @@ const Dropdown: React.FC<DropdownProps> = (props) => {
         <EnsembleFormItem values={values}>
           <Select
             className={`${values?.styles?.names || ""} ${id}_input`}
+            defaultValue={values?.selectedValue}
```

Comment:
This should just be `values?.value`. `selectedValue` is fine for internal variable name but the form item interface always uses `value`